js-callback-functions/js-callback-functions_user-login-callback-2-with-argument/js/index.js

```diff
@@ -5,3 +5,16 @@ function handleUserLogin(onSuccess) {
 }
 
 // The exercise starts here!
+
+// Create function "showWelcomeMessage"
+function showWelcomeMessage(userName) {
+  console.log(`Welcome ${userName}! You are logged in now.`);
+}
+
+// Call handleUserLogin with callback function
+handleUserLogin(showWelcomeMessage);
+
+// Call handleUserLogin with anonymous function as callback
+handleUserLogin(function (userName) {
```

Comment:
works also without writing "function" but fine like that as well

Reply:
Then I would need to write the arrows for an arrow function too.

Reply:
ups scusi, thought there was one